docs/class-definitions.md

```diff
@@ -154,66 +162,122 @@ await space.updateProfileData((oldProfileData) => {
 })
 ```
 
+# Members
+
+Handles members within a space.
+
+## Methods
+
 ### subscribe
 
-Listen to events for the space. See [EventEmitter](/docs/usage.md#event-emitters) for overloading usage.
+Listen to member events for the space. See [EventEmitter](/docs/usage.md#event-emitters) for overloading usage.
 
 Available events:
 
-- #### **membersUpdate**
+- #### **update**
 
-  Listen to updates to members.
+  Listen to all updates for members in a space.
 
   ```ts
-  space.subscribe('membersUpdate', (members: SpaceMember[]) => {});
+  space.members.subscribe('update', (member: SpaceMember) => {});
+  ```
+
+  This is the same as not specifying the event:
+
+  ```ts
+  space.members.subscribe((member: SpaceMember) => {})
+
   ```
 
   Triggers on:
-  - presence updates ([`enter`, `leave`, `update` and `present` events](https://ably.com/docs/presence-occupancy/presence?lang=javascript))
+  - presence updates ([`enter`, `leave`, `remove`, `update` events](https://ably.com/docs/presence-occupancy/presence?lang=javascript))
   - [location updates](#locationupdate)
```

Comment:
As per [this code comment](https://github.com/ably-labs/spaces/blob/main/src/Members.ts#L55-L56), the member `update` event does _not_ fire for location updates, only profile data updates.